pkg/flag/flag.go

```diff
@@ -26,7 +26,7 @@ const (
 var (
 	flagSet *pflag.FlagSet
 
-	TektonVersion   = "v0.7.0"
+	TektonVersion   = "v0.8.0"
```

Comment:
this should be a flag instead .. "latest" by default so that we don't have to keep changing it

Reply:
That will be an improvement. Let us add it as an improvement/tech-debt story in jira.